<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="14" failed="0" total="22" passed="8">
  <reporter-output>
  </reporter-output>
  <suite name="Test Suites" duration-ms="0" started-at="2015-10-27T15:51:28Z" finished-at="2015-10-27T15:51:28Z">
    <groups>
    </groups>
  </suite> <!-- Test Suites -->
  <suite name="Suite1" duration-ms="1966" started-at="2015-10-27T15:51:25Z" finished-at="2015-10-27T15:51:27Z">
    <groups>
    </groups>
    <test name="Test A1" duration-ms="1080" started-at="2015-10-27T15:51:25Z" finished-at="2015-10-27T15:51:26Z">
      <class name="suite1.TestA1">
        <test-method status="SKIP" signature="testA1(java.util.Hashtable)[pri:0, instance:suite1.TestA1@561f72d0]" name="testA1" duration-ms="127" started-at="2015-10-27T21:21:26Z" data-provider="getDataSuite1" finished-at="2015-10-27T21:21:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, TestData=Login1, Iteration=1.0, username=sdfdsf, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test : TestA1 as the Runmode of Test Case TestA1 is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test : TestA1 as the Runmode of Test Case TestA1 is No
	at Utility.CommonUtils.checkExecution(CommonUtils.java:85)
	at suite1.TestA1.testA1(TestA1.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testA1 -->
        <test-method status="SKIP" signature="testA1(java.util.Hashtable)[pri:0, instance:suite1.TestA1@561f72d0]" name="testA1" duration-ms="83" started-at="2015-10-27T21:21:26Z" data-provider="getDataSuite1" finished-at="2015-10-27T21:21:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, TestData=Login2, Iteration=2.0, username=sdfsd, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test : TestA1 as the Runmode of Test Case TestA1 is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test : TestA1 as the Runmode of Test Case TestA1 is No
	at Utility.CommonUtils.checkExecution(CommonUtils.java:85)
	at suite1.TestA1.testA1(TestA1.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testA1 -->
      </class> <!-- suite1.TestA1 -->
    </test> <!-- Test A1 -->
    <test name="Test A2" duration-ms="486" started-at="2015-10-27T15:51:26Z" finished-at="2015-10-27T15:51:27Z">
      <class name="suite1.TestA2">
        <test-method status="PASS" signature="testA2(java.util.Hashtable)[pri:0, instance:suite1.TestA2@3210a146]" name="testA2" duration-ms="104" started-at="2015-10-27T21:21:26Z" data-provider="getDataSuite1" finished-at="2015-10-27T21:21:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, TestData=SignUp1, Iteration=1.0, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testA2 -->
        <test-method status="PASS" signature="testA2(java.util.Hashtable)[pri:0, instance:suite1.TestA2@3210a146]" name="testA2" duration-ms="80" started-at="2015-10-27T21:21:27Z" data-provider="getDataSuite1" finished-at="2015-10-27T21:21:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, TestData=SignUp2, Iteration=2.0, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testA2 -->
        <test-method status="PASS" signature="testA2(java.util.Hashtable)[pri:0, instance:suite1.TestA2@3210a146]" name="testA2" duration-ms="74" started-at="2015-10-27T21:21:27Z" data-provider="getDataSuite1" finished-at="2015-10-27T21:21:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, TestData=SignUp3, Iteration=3.0, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testA2 -->
        <test-method status="SKIP" signature="testA2(java.util.Hashtable)[pri:0, instance:suite1.TestA2@3210a146]" name="testA2" duration-ms="68" started-at="2015-10-27T21:21:27Z" data-provider="getDataSuite1" finished-at="2015-10-27T21:21:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=N, TestData=SignUp4, Iteration=4.0, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test : TestA2 as the Runmode of Test Data is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test : TestA2 as the Runmode of Test Data is No
	at Utility.CommonUtils.checkExecution(CommonUtils.java:90)
	at suite1.TestA2.testA2(TestA2.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testA2 -->
        <test-method status="SKIP" signature="testA2(java.util.Hashtable)[pri:0, instance:suite1.TestA2@3210a146]" name="testA2" duration-ms="84" started-at="2015-10-27T21:21:27Z" data-provider="getDataSuite1" finished-at="2015-10-27T21:21:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=N, TestData=SignUp5, Iteration=5.0, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test : TestA2 as the Runmode of Test Data is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test : TestA2 as the Runmode of Test Data is No
	at Utility.CommonUtils.checkExecution(CommonUtils.java:90)
	at suite1.TestA2.testA2(TestA2.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testA2 -->
      </class> <!-- suite1.TestA2 -->
    </test> <!-- Test A2 -->
    <test name="Test A3" duration-ms="363" started-at="2015-10-27T15:51:27Z" finished-at="2015-10-27T15:51:27Z">
      <class name="suite1.TestA3">
        <test-method status="SKIP" signature="testA3(java.util.Hashtable)[pri:0, instance:suite1.TestA3@2bc63b78]" name="testA3" duration-ms="86" started-at="2015-10-27T21:21:27Z" data-provider="getDataSuite1" finished-at="2015-10-27T21:21:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=N, TestData=UserReg1, Iteration=1.0, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test : TestA3 as the Runmode of Test Case TestA3 is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test : TestA3 as the Runmode of Test Case TestA3 is No
	at Utility.CommonUtils.checkExecution(CommonUtils.java:85)
	at suite1.TestA3.testA3(TestA3.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testA3 -->
        <test-method status="SKIP" signature="testA3(java.util.Hashtable)[pri:0, instance:suite1.TestA3@2bc63b78]" name="testA3" duration-ms="87" started-at="2015-10-27T21:21:27Z" data-provider="getDataSuite1" finished-at="2015-10-27T21:21:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=N, TestData=UserReg2, Iteration=2.0, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test : TestA3 as the Runmode of Test Case TestA3 is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test : TestA3 as the Runmode of Test Case TestA3 is No
	at Utility.CommonUtils.checkExecution(CommonUtils.java:85)
	at suite1.TestA3.testA3(TestA3.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testA3 -->
        <test-method status="SKIP" signature="testA3(java.util.Hashtable)[pri:0, instance:suite1.TestA3@2bc63b78]" name="testA3" duration-ms="75" started-at="2015-10-27T21:21:27Z" data-provider="getDataSuite1" finished-at="2015-10-27T21:21:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, TestData=UserReg3, Iteration=3.0, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test : TestA3 as the Runmode of Test Case TestA3 is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test : TestA3 as the Runmode of Test Case TestA3 is No
	at Utility.CommonUtils.checkExecution(CommonUtils.java:85)
	at suite1.TestA3.testA3(TestA3.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testA3 -->
        <test-method status="SKIP" signature="testA3(java.util.Hashtable)[pri:0, instance:suite1.TestA3@2bc63b78]" name="testA3" duration-ms="56" started-at="2015-10-27T21:21:27Z" data-provider="getDataSuite1" finished-at="2015-10-27T21:21:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, TestData=UserReg4, Iteration=4.0, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test : TestA3 as the Runmode of Test Case TestA3 is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test : TestA3 as the Runmode of Test Case TestA3 is No
	at Utility.CommonUtils.checkExecution(CommonUtils.java:85)
	at suite1.TestA3.testA3(TestA3.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testA3 -->
      </class> <!-- suite1.TestA3 -->
    </test> <!-- Test A3 -->
  </suite> <!-- Suite1 -->
  <suite name="Suite2" duration-ms="817" started-at="2015-10-27T15:51:27Z" finished-at="2015-10-27T15:51:28Z">
    <groups>
    </groups>
    <test name="Test B1" duration-ms="170" started-at="2015-10-27T15:51:27Z" finished-at="2015-10-27T15:51:27Z">
      <class name="suite2.TestB1">
        <test-method status="PASS" signature="testB1(java.util.Hashtable)[pri:0, instance:suite2.TestB1@6d5bd16]" name="testB1" duration-ms="60" started-at="2015-10-27T21:21:27Z" data-provider="getDataSuite2" finished-at="2015-10-27T21:21:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, TestData=Login1, Iteration=1.0, username=sdfdsf, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testB1 -->
        <test-method status="PASS" signature="testB1(java.util.Hashtable)[pri:0, instance:suite2.TestB1@6d5bd16]" name="testB1" duration-ms="57" started-at="2015-10-27T21:21:27Z" data-provider="getDataSuite2" finished-at="2015-10-27T21:21:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, TestData=Login2, Iteration=2.0, username=sdfsd, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testB1 -->
      </class> <!-- suite2.TestB1 -->
    </test> <!-- Test B1 -->
    <test name="Test B2" duration-ms="380" started-at="2015-10-27T15:51:27Z" finished-at="2015-10-27T15:51:28Z">
      <class name="suite2.TestB2">
        <test-method status="PASS" signature="testB2(java.util.Hashtable)[pri:0, instance:suite2.TestB2@6c2e1e17]" name="testB2" duration-ms="75" started-at="2015-10-27T21:21:27Z" data-provider="getDataSuite2" finished-at="2015-10-27T21:21:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, TestData=SignUp1, Iteration=1.0, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testB2 -->
        <test-method status="PASS" signature="testB2(java.util.Hashtable)[pri:0, instance:suite2.TestB2@6c2e1e17]" name="testB2" duration-ms="85" started-at="2015-10-27T21:21:28Z" data-provider="getDataSuite2" finished-at="2015-10-27T21:21:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, TestData=SignUp2, Iteration=2.0, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testB2 -->
        <test-method status="PASS" signature="testB2(java.util.Hashtable)[pri:0, instance:suite2.TestB2@6c2e1e17]" name="testB2" duration-ms="75" started-at="2015-10-27T21:21:28Z" data-provider="getDataSuite2" finished-at="2015-10-27T21:21:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, TestData=SignUp3, Iteration=3.0, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testB2 -->
        <test-method status="SKIP" signature="testB2(java.util.Hashtable)[pri:0, instance:suite2.TestB2@6c2e1e17]" name="testB2" duration-ms="57" started-at="2015-10-27T21:21:28Z" data-provider="getDataSuite2" finished-at="2015-10-27T21:21:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=N, TestData=SignUp4, Iteration=4.0, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test : TestB2 as the Runmode of Test Data is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test : TestB2 as the Runmode of Test Data is No
	at Utility.CommonUtils.checkExecution(CommonUtils.java:90)
	at suite2.TestB2.testB2(TestB2.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testB2 -->
        <test-method status="SKIP" signature="testB2(java.util.Hashtable)[pri:0, instance:suite2.TestB2@6c2e1e17]" name="testB2" duration-ms="40" started-at="2015-10-27T21:21:28Z" data-provider="getDataSuite2" finished-at="2015-10-27T21:21:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=N, TestData=SignUp5, Iteration=5.0, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test : TestB2 as the Runmode of Test Data is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test : TestB2 as the Runmode of Test Data is No
	at Utility.CommonUtils.checkExecution(CommonUtils.java:90)
	at suite2.TestB2.testB2(TestB2.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testB2 -->
      </class> <!-- suite2.TestB2 -->
    </test> <!-- Test B2 -->
    <test name="Test B3" duration-ms="250" started-at="2015-10-27T15:51:28Z" finished-at="2015-10-27T15:51:28Z">
      <class name="suite2.TestB3">
        <test-method status="SKIP" signature="testB3(java.util.Hashtable)[pri:0, instance:suite2.TestB3@62d438fd]" name="testB3" duration-ms="57" started-at="2015-10-27T21:21:28Z" data-provider="getDataSuite2" finished-at="2015-10-27T21:21:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=N, TestData=UserReg1, Iteration=1.0, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test : TestB3 as the Runmode of Test Case TestB3 is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test : TestB3 as the Runmode of Test Case TestB3 is No
	at Utility.CommonUtils.checkExecution(CommonUtils.java:85)
	at suite2.TestB3.testB3(TestB3.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testB3 -->
        <test-method status="SKIP" signature="testB3(java.util.Hashtable)[pri:0, instance:suite2.TestB3@62d438fd]" name="testB3" duration-ms="73" started-at="2015-10-27T21:21:28Z" data-provider="getDataSuite2" finished-at="2015-10-27T21:21:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=N, TestData=UserReg2, Iteration=2.0, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test : TestB3 as the Runmode of Test Case TestB3 is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test : TestB3 as the Runmode of Test Case TestB3 is No
	at Utility.CommonUtils.checkExecution(CommonUtils.java:85)
	at suite2.TestB3.testB3(TestB3.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testB3 -->
        <test-method status="SKIP" signature="testB3(java.util.Hashtable)[pri:0, instance:suite2.TestB3@62d438fd]" name="testB3" duration-ms="43" started-at="2015-10-27T21:21:28Z" data-provider="getDataSuite2" finished-at="2015-10-27T21:21:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, TestData=UserReg3, Iteration=3.0, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test : TestB3 as the Runmode of Test Case TestB3 is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test : TestB3 as the Runmode of Test Case TestB3 is No
	at Utility.CommonUtils.checkExecution(CommonUtils.java:85)
	at suite2.TestB3.testB3(TestB3.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testB3 -->
        <test-method status="SKIP" signature="testB3(java.util.Hashtable)[pri:0, instance:suite2.TestB3@62d438fd]" name="testB3" duration-ms="44" started-at="2015-10-27T21:21:28Z" data-provider="getDataSuite2" finished-at="2015-10-27T21:21:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{RunMode=Y, TestData=UserReg4, Iteration=4.0, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test : TestB3 as the Runmode of Test Case TestB3 is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test : TestB3 as the Runmode of Test Case TestB3 is No
	at Utility.CommonUtils.checkExecution(CommonUtils.java:85)
	at suite2.TestB3.testB3(TestB3.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testB3 -->
      </class> <!-- suite2.TestB3 -->
    </test> <!-- Test B3 -->
  </suite> <!-- Suite2 -->
</testng-results>
